sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/PythonService.java

```diff
@@ -40,25 +40,43 @@ public class PythonService {
 
   private final String module;
   private final List<String> args;
+  private final List<String> extraPackages;
 
-  public PythonService(String module, List<String> args) {
+  public PythonService(String module, List<String> args, List<String> extraPackages) {
     this.module = module;
     this.args = args;
+    this.extraPackages = extraPackages;
+  }
+
+  public PythonService(String module, List<String> args) {
+    this(module, args, ImmutableList.of());
   }
 
   public PythonService(String module, String... args) {
     this(module, Arrays.asList(args));
   }
 
+  public PythonService withExtraPackages(List<String> extraPackages) {
```

Comment:
Also add a Java doc (but this can be less details since this is mostly an internal utility).

Reply:
Done.